### PR TITLE
Hive insert statements

### PR DESCRIPTION
**Pull request by rukmini-g**

### CODE REVIEW

File: git/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Testing to raise PR
```
Comment (animenon) [inline git/test.txt]: Delete this file.

File: hive insert statements [hunk 2]
```diff
@@ -0,0 +1,28 @@
+''' INSERT INTO TABLE table_name SELECT columnlist FROM secondtable; '''
```
Comment (animenon) [inline hive insert statements]: Make this a `.md` file and add appropriate Headings.

**Review by animenon — CHANGES_REQUESTED** [review]

File: hive insert statements.md [hunk 3]
```diff
@@ -1,28 +1,34 @@
-```INSERT INTO TABLE table_name SELECT columnlist FROM secondtable;```
```
Comment (animenon) [inline hive insert statements.md]: Please add a header.

File: hive insert statements.md [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
+ they do not go through the HDFS trash mechanism.*
 
 ``` INSERT INTO TABLE table_name SELECT * FROM some_other_table; ```
     
 ``` create table t1 ( i int ) partitioned by (x int , y string); ```
 ``` INSERT INTO t1 PARTITION ( x=10, y=’a’ ) SELECT c1 FROM some_other table; ```
-#*All inserted rows will have the same x and y values. This technique of specifiying all the partition key values is known as static partitioning.*
-*If you want to use Static partition in Hive you should set property 
+
+#*All inserted rows will have the same x and y values. This technique of specifiying all the partition key values is known as static           partitioning.*
```
Comment (animenon) [inline hive insert statements.md]: Subheadings should use `### ` not `# `

**Review by animenon — CHANGES_REQUESTED** [review]

File: hive insert statements.md [hunk 5]
```diff
@@ -9,26 +9,26 @@
...
     
 ``` create table t1 ( i int ) partitioned by (x int , y string); ```
 ``` INSERT INTO t1 PARTITION ( x=10, y=’a’ ) SELECT c1 FROM some_other table; ```
 
-#*All inserted rows will have the same x and y values. This technique of specifiying all the partition key values is known as static           partitioning.*
+#### *All inserted rows will have the same x and y values. This technique of specifiying all the partition key values is known as static           partitioning.*
 
-#*If you want to use Static partition in Hive you should set property 
+#### *If you want to use Static partition in Hive you should set property 
      ``` set hive.mapred.mode = strict ``` This property set by default in hive-site.xml.*
```
Comment (animenon) [inline hive insert statements.md]: This doesn't look right, why bold? and why is the line `"This property set ..."` is on the same line?

File: hive insert statements.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-
+# Hive insert statements
```
Comment (animenon) [inline hive insert statements.md]: This is right.

**Comment by rukmini-g** [comment]
closed

**Pull request closed without merging**